README.md

```diff
@@ -1,5 +1,10 @@
 # Flutter SDK
 
+> [!NOTE]  
+> #### Updates for Billing Library 5
+> - SDK Version 1.0: This version uses Google Billing Library 5.2.1 APIs to fetch product information from the Google Play Console and make purchases. If you’re integrating Chargebee’s SDK for the first time, then use this version, and if you’re migrating from the older version of SDK to this version, follow the migration steps in this [document](https://www.chargebee.com/docs/2.0/mobile-playstore-billing-library-5.html).
```

Comment:
SDK Version 1.0 - Do u wish to add beta here as well? @cb-haripriyan

Reply:
We can update this.